usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1129,7 +1129,7 @@
                 ("recipient_foreign_city_name", "transaction__assistance_data__legal_entity_foreign_city"),
                 ("recipient_foreign_province_name", "transaction__assistance_data__legal_entity_foreign_provi"),
                 ("recipient_foreign_postal_code", "transaction__assistance_data__legal_entity_foreign_posta"),
-                ("primary_place_of_performance_scope", "transaction__assistance_data__place_of_performance_scope",),
```

Comment:
Did this work before?

Reply:
In Python, `(1, 2) == (1, 2, )`.  I'm sure if you looked you'd find a long winded, well reasoned explanation as to why that is.  Perhaps because it's the literal way to create tuples with one constituient?  `(1, )`

I'm not sure what fixed this.  I'm pretty sure I didn't.  That seems a bit too obscure to grab my attention.  Perhaps Black?